src/pages/Login.tsx

```diff
@@ -23,38 +24,48 @@ const styles = {
 function Login() {
     const [showQR, setShowQR] = useState<string>();
```

Comment:
is this the QR string? can we rename to QRstring or QRpayload to make more clear?

Reply:
a QR payload is always a string